indico/htdocs/sass/partials/reports.scss

```diff
@@ -48,6 +50,15 @@
                 display: inline-block;
             }
         }
+
+        td {
+            vertical-align: top;
+
+            .vertical-aligner {
+                display: block;
+                padding-top: 0.27em;  // Ensure that single line text cells are aligned in the middle
```

Comment:
Well, the comment explains it, I had to put a top padding that will have the same effect as the middle vertical alignment for single rows so that when the line expands to a bigger height, the elements inside will still be symmetrical.